examples/exports/index.mjs

```diff
@@ -0,0 +1,31 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+
+(async () => {
+  const stdlib = await loadStdlib();
+  const startingBalance = stdlib.parseCurrency(100);
+
+  const alice = await stdlib.newTestAccount(startingBalance);
+  const bob = await stdlib.newTestAccount(startingBalance);
+
+  const ctcAlice = alice.deploy(backend);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+
+  console.log('Exports:');
+  const exports = backend.getExports(stdlib);
+  const a = exports.a;
+  console.log(`a: ${JSON.stringify(a)}`);
+  const o = exports.o;
+  console.log(`o: ${JSON.stringify(o)}`);
```

Comment:
I think you can do `===` and check that it looks like what we know its supposed to be so this is really a test case

Reply:
I added an equality check for the exports/result of functions.